scripts/env.sh

```diff
@@ -3,7 +3,7 @@
 cd "$(cd -P -- "$(dirname -- "$0")" && pwd -P)" || exit 2
 
 # Binary paths
-export PATH="$PWD/bin:$PWD/pgsql/bin:$PWD/node/bin:$PATH"
+export PATH="$PWD/bin:$PWD/pgsql/bin:$PWD/node/bin:$PWD/lisky/bin:$PWD/pm2/bin$PATH"
```

Comment:
Missing a colon

Reply:
But actually global npm installation should add the lisky executable to path anyway.

Reply:
nice catch thank you :

Reply:
No. You are misunderstanding what this does for us.

env.sh creates a fake environment setup to store all binary stuff in one place without touching the OS.